### ISSUE TITLE
Unable to use with Swift 5.6 on Ubuntu 20.04

### ISSUE THREAD

**Issue by PierceNg** [moshegottlieb/SwiftSQLite#1]
```
% cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.4 LTS"

% swift --version
Swift version 5.6.1 (swift-5.6.1-RELEASE)
Target: x86_64-unknown-linux-gnu

% swift package update
Updating https://github.com/moshegottlieb/SwiftSQLite
Updating https://github.com/apple/swift-argument-parser
Updated https://github.com/apple/swift-argument-parser (0.54s)
Updated https://github.com/moshegottlieb/SwiftSQLite (0.54s)
Computing version for https://github.com/moshegottlieb/SwiftSQLite
Computed https://github.com/moshegottlieb/SwiftSQLite at 1.0.51 (0.02s)
Computing version for https://github.com/apple/swift-argument-parser
Computed https://github.com/apple/swift-argument-parser at 1.1.2 (0.03s)
Everything is already up-to-date
'swiftsqlite': error: Source files for target SQLite3 should be located under 'Sources/SQLite3', or a custom sources path can be set with the 'path' property in Package.swift
```

**Comment by moshegottlieb** [comment]
Sorry about that :-)  
Fixed it in `v1.0.52`.  
Please note that swift SPM does not install dependencies, so you still need to install the sqlite3 development package on your build machine.  
You're on ubuntu so it should be:  
`sudo apt-get install libsqlite3-dev`

Thanks for reporting!

**Comment by PierceNg** [comment]
Thanks for the quick response!
